.github/workflows/ci.yaml

```diff
@@ -772,6 +772,7 @@ jobs:
           flux --namespace flux-system reconcile kustomization coder
           flux --namespace flux-system reconcile source chart coder-coder
```

Comment:
needs coder-coder-provisioner (or whatever it is from scripts/reconcile.sh) here too